website/docs/admin_hosts_maya.md

```diff
@@ -110,6 +106,35 @@ or Deadlines **Draft Tile Assembler**.
 This is useful to fix some specific renderer glitches and advanced hacking of Maya Scene files. `Patch name` is label for patch for easier orientation.
 `Patch regex` is regex used to find line in file, after `Patch line` string is inserted. Note that you need to add line ending.
 
+### Extract GPU Cache
+
+![Maya GPU Cache](assets/maya-admin_gpu_cache.png)
+
+- **Evaluate Every** Specifies which samples are saved during cache creation. For example, a value of 2 specifies that only every other sample specified by the Evaluate every # frame(s) option is saved to your Alembic file.
+
+- **Save Every** Specifies how often samples are taken during file creation. By default, one sample of your object's transformations is taken every frame and saved to the Alembic file.
```

Comment:
It feels like the descriptions here are reversed? **Evaluate Every** talks about **Save Every** and save talks about taking samples. Are these correct?

Reply:
> **Step**
> 
> Specifies how often samples are taken during file creation. By default, one sample of your object's transformations is taken every frame and saved to the Alembic file.
> For example, a value of 2 caches the transformations of the current object at every other frame of the Cache Time Range.
>
> **Save every # evaluation(s)**
>     Specifies which samples are saved during cache creation. For example, a value of 2 specifies that only every other sample specified by the Evaluate every # frame(s) option is saved to your Alembic file.

So this does look swapped around, right?

Reply:
Yeah, confusing all around. Have change it to `step` and `stepSave`.